app/handlers/dbindexes.py

```diff
@@ -320,6 +321,33 @@ def _ensure_test_group_indexes(database):
     )
 
 
+def _ensure_test_group_regressions_indexes(database):
```

Comment:
I thought this was supposed to be in #98? Can you rebase without these index patches, and make sure #98 is up-to-date so we can merge both PRs?

Reply:
This two new collections are added in this PR (see the first commit), while #98 is for fixing the missing boot regressions indexes.

Reply:
Ah yes sorry, my bad.  So I'll check #98 again and merge it if it looks ready as it's been tested already.